src/HTML.js

```diff
@@ -33,7 +33,7 @@ export default class HTML extends PureComponent {
         uri: PropTypes.string,
         tagsStyles: PropTypes.object,
         classesStyles: PropTypes.object,
-        containerStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes.style,
+        containerStyle:  ViewPropTypes ? ViewPropTypes.style : PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
I suggest you get rid of the ternary operations and directly assign the loose `PropTypes.oneOfType([PropTypes.object, PropTypes.array])`, because we don't really need to keep support for something that will eventually disappear.